app.js

```diff
@@ -1,13 +1,35 @@
 const express = require("express");
 const app = express();
 const { getTopics } = require("./controllers/topics.controller");
+const { getArticleById } = require("./controllers/articles.controller");
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById);
 
+//404 for end points not found
 app.use((req, res, next) => {
```

Comment:
This'll work and is fine, but maybe have a look over the differences between app.use and app.all to achieve what you're doing here.